### PR TITLE
Add fortify job

### PR DESCRIPTION
**Pull request by eventuri**

### CODE REVIEW

File: lib/gitlab/ci/templates/pipeline/GradleJavaPipeline.yml [hunk 1]
```diff
@@ -39,6 +40,7 @@ include:
   - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/release/1.3/lib/gitlab/ci/templates/jobs/gradle/DependencyScanning.yml
   - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/release/1.3/lib/gitlab/ci/templates/jobs/gradle/LicenseScanning.yml
   - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/release/1.3/lib/gitlab/ci/templates/jobs/gradle/StaticApplicationSecurityTesting.yml
+  - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/add-fortify-job/lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml
```
Comment (eventuri) [inline lib/gitlab/ci/templates/pipeline/GradleJavaPipeline.yml]: "add-fortify-job" needs to be changed to "release/1.3" before merging.

File: lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  KUBERNETES_HELPER_MEMORY_REQUEST: 4G
+  KUBERNETES_HELPER_MEMORY_LIMIT: 4G
+
+fortify_scanning:
+  stage: test
+  image: ${TAK_REGISTRY_HOST}/devsecops/fortify-tooling:1.2.0
+  allow_failure: true
+  script:
+    - set -x
+    - fortifyupdate && sourceanalyzer -Xmx4096M -b $CI_JOB_NAME -clean && sourceanalyzer -Xmx4096M -b $CI_JOB_NAME -project-root ./.fortify -logfile "./${CI_JOB_NAME}-analyze.log" -exclude "./src/test/**/*.*" -source $JAVA_SRC_VERSION $PACKAGE_ENTRY_POINT && sourceanalyzer -Xmx4096M -b $CI_JOB_NAME -scan -project-root ./.fortify -logfile "./${CI_JOB_NAME}.txt" -f "./${CI_JOB_NAME}-scan_results.fpr"
```
Comment (eventuri) [inline lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml]: Should move '-exclude ".src/test/**/*.*"' to something that is more configurable.

Comment (eventuri) [inline lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml]: Should add variable to allow optional passing of a "-rules" file.

**Review by AndrewTrow — APPROVED** [review]

File: lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  PACKAGE_ENTRY_POINT: "./**/*.java"
+  KUBERNETES_HELPER_MEMORY_REQUEST: 4G
+  KUBERNETES_HELPER_MEMORY_LIMIT: 4G
+  FORTIFY_EXCLUDE_FLAGS: -exclude "./src/test/**/*.*"
+  # The Fortify rules flags can be something like "-rules rules.xml"
+  FORTIFY_RULES_FLAGS: ""
+
+fortify_scanning:
+  stage: test
+  image: ${TAK_REGISTRY_HOST}/devsecops/fortify-tooling:1.2.0
```
Comment (AndrewTrow) [inline lib/gitlab/ci/templates/jobs/security/FortifyScanning.yml]: Can we make this a variable?  JEACO can use this template if we could set it

**Review by AndrewTrow — APPROVED** [review]

**Pull request merged**